tools/bash/BUILD.tools

```diff
@@ -0,0 +1,6 @@
+exports_files(["bash_def.bzl"])
+
+toolchain_type(
+    name = "bash_toolchain_type",
```

Comment:
For consistency with other toolchains (and to save typing) I recommend naming this simply "toolchain_type".